### PR TITLE
`epm serve`

### PR DESCRIPTION
**Pull request by compleatang**
builds a servable interface to most of the epm command line for work from remotes

### CODE REVIEW

File: server/httpService.go [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+	errString := fmt.Sprintf("%v", err)
+	logger.Errorf("ERROR :(\treturning http code: %v\tbecause: %s\n", code, errString)
+	this.writeMsg(w, code, errString)
+}
+
+// Helper function to execute the cli commands and send back
+// the result of the command to the caller.
+func (this *HttpService) executeCommand(cmdRaw []string, w http.ResponseWriter) {
+	product, err := this.executeCommandRaw(cmdRaw, w)
+	if err != nil {
```
Comment (ebuchman) [inline server/httpService.go]: why don't we return/log the error (from Stderr in executeCommandRaw)?

File: server/server.go [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+an empty string then the server will use the request body
+string as the genesis.json.
+
+Optional Parameters:
+
+	- checkout = if checkout=false then the fetched blockchain will not be checked out.
+	- type = instatiate a non-thelonious chain which is part of EPM's capabilities.
+
+Note, there is no check that the passed string is valid json
+before the server will seek to instatiate the chain.
```
Comment (ebuchman) [inline server/server.go]: So if it fails we get the default, ya?

Comment (compleatang) [inline server/server.go]: yes.

File: server/httpService.go (hunk 1 above)
Comment (compleatang) [inline server/httpService.go]: cause nothing was coming from stderr. have added now.

**Pull request merged**